src/poprox_recommender/nrms.py

```diff
@@ -230,15 +215,30 @@ def select_articles(
         token_mapping,
     )
 
+    # Load model
+    model = load_model(model_checkpoint, model_device)
+
+    # Compute today's article similarity matrix
```

Comment:
This chunk of code that computes the article similarity matrix would make a good function. Maybe something like `compute_similarity_matrix()`?

Reply:
Make sense, refactored!